### PR TITLE
Fixed the broken link

### PR DESCRIPTION
**Pull request by mayank7**
Fixed broken links in responses in DuckDuckGo.

IA Page: https://duck.co/ia/view/about_duckduckgo

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [About Duckduckgo](https://duck.co/ia/view/about_duckduckgo)

**Description**: Details about DuckDuckGo

**Example Query**: [https://duckduckgo.com/?q=about+duckduckgo&ia=answer](https://beta.duckduckgo.com/?q=https://duckduckgo.com/?q=about+duckduckgo&ia=answer), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/about_duckduckgo)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/about_duckduckgo)*

**Comment by mintsoft** [comment]
Thanks @mayank7

File: t/DuckDuckGo.t [hunk 1]
```diff
@@ -97,14 +97,14 @@ my @merch_result = (
         }
     });
 my @tor_result = (
-    'DuckDuckGo is available on Tor http://3g2upl4pq6kufc4m.onion',
+    'DuckDuckGo is available on Tor http://3g2upl4pq6kufc4m.onion.link',
```
Comment (moollaza) [inline t/DuckDuckGo.t]: I'm not sure we want the `.link` here? Will those still work in Tor Browser?

**Comment by mayank7** [comment]
@moollaza I checked without .link, it was not working in Chrome and Firefox.
Adding .link made it work in both, so I added.

**Comment by moollaza** [comment]
@mayank7 did you make the about_duckduckgo IA Page?

This should be using https://duck.co/ia/view/duck_duck_go

**Comment by mayank7** [comment]
@moollaza I din't create that page, I just used it.

**Comment by moollaza** [comment]
@mayank7 thanks for clarifying! And thanks again for fixing the DuckDuckGo Goodie! You did a great job. We at DuckDuckGo, the DuckDuckHack community, and all the people who will encounter this Instant Answer really appreciate it.

We could really use your help on something else now. We want to increase our Instant Answer coverage for as many programming languages and platforms as possible. We want a Haskell Cheat Sheet and I think you'd similarly do a great job with building that. Please update the [Programming IA Coverage table](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage) (mark it with a 'd') if you'd be willing to take it up!

I think it is a good fit for you, but I'm of course not entirely sure, so I'd love to chat if you're around this week or next. Please message me on slack (@moollaza) or send me an email (moollaza@duckduckgo.com) if you'd be interested in talking. 

Thanks again!

**Comment by mayank7** [comment]
@moollaza No problem. Bdw, I am already on slack.

**Pull request merged**